.circleci/config.yml

```diff
@@ -32,7 +32,14 @@ jobs:
 
             - run: yarn lint
 
-            - run: yarn test --ci --maxWorkers=2 --coverage --coverageReporters=text-lcov | ./node_modules/coveralls/bin/coveralls.js
+            - run: yarn test --ci --maxWorkers=2 --coverage --coverageReporters=text-lcov
+
+            - run:
+                  name: run coveralls
+                  command: |
+                      if [ -n "$COVERALLS_REPO_TOKEN" ]; then
+                          cat coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
`npx coveralls` should work as well instead of `./node_modules/coveralls/bin/coveralls.js`